### ISSUE TITLE
Multiple sizes

### ISSUE THREAD

**Issue by own2pwn** [val92130/Supreme-Auto-Checkout#59]
Hey there! You've created a really nice bot, thanks for it! 
I'd like to know what do you think about adding the possibility to choose several sizes for items / category?
I've seen code a little, can't we create a helper function that will accept array of sizes to compare with, do `for each size in sizes {}` and return early if we've a match?

**Comment by own2pwn** [comment]
I think you got the idea, but if you need pseudo-code just ping me

**Comment by val92130** [comment]
That's a very good idea, thanks!

I'll try to implement this, it shouldn't be really difficult :)

**Comment by own2pwn** [comment]
Also wouldn't it be cool to have per-item sizing too? E.g. I want hoodie A in S, hoodie B in L, and any other in M. But what do you think about performance? Won't it slow bot a little?

**Comment by val92130** [comment]
You're right it would be cool! But no don't worry it won't slow down the bot, I'll add this in my todo list for the next update :)

**Comment by own2pwn** [comment]
Just not to start new issue, how do I cop camo headband? What if I place `camo` in color? It will add random from yellow/white? Or color should be exact?

**Comment by val92130** [comment]
The color has to be somewhat similar, for example if the color is `red camo` and you put `camo` it will work, and you can add multiple colors like `camo black yellow` it will take the first matching color.

**Comment by own2pwn** [comment]
Maybe someone(native speakers) could help with creating wiki page with common questions like this one about colors? It should be a great PR for free bot!
